manager/test/api/src/test/resources/test-plan-data/import/001_import.resttest

```diff
@@ -1,403 +1,403 @@
+          "entityId": "Client1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973206961,
+          "what": "Create"
+        },
+        {
+          "id": 9,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Api",
+          "entityId": "API1",
+          "createdOn": 1445973207033,
+          "what": "Create"
+        },
+        {
+          "id": 11,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Api",
+          "entityId": "API1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207101,
+          "what": "Create"
+        },
+        {
+          "id": 12,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Api",
+          "entityId": "API1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207198,
+          "what": "Update",
+          "data": "{\"changes\":[{\"name\":\"plans\",\"before\":\"\",\"after\":\"Plan1:1.0\"},{\"name\":\"endpoint\",\"before\":null,\"after\":\"http://localhost:8080/ping\"},{\"name\":\"endpointType\",\"before\":null,\"after\":\"rest\"}]}"
+        },
+        {
+          "id": 13,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Api",
+          "entityId": "API1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207594,
+          "what": "Publish"
+        },
+        {
+          "id": 15,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Client",
+          "entityId": "Client1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207623,
+          "what": "CreateContract",
+          "data": "{\"clientOrgId\":\"Organization1\",\"clientId\":\"Client1\",\"clientVersion\":\"1.0\",\"apiOrgId\":\"Organization1\",\"apiId\":\"API1\",\"apiVersion\":\"1.0\",\"planId\":\"Plan1\",\"planVersion\":\"1.0\"}"
+        },
+        {
+          "id": 16,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Api",
+          "entityId": "API1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207628,
+          "what": "CreateContract",
+          "data": "{\"clientOrgId\":\"Organization1\",\"clientId\":\"Client1\",\"clientVersion\":\"1.0\",\"apiOrgId\":\"Organization1\",\"apiId\":\"API1\",\"apiVersion\":\"1.0\",\"planId\":\"Plan1\",\"planVersion\":\"1.0\"}"
+        },
+        {
+          "id": 17,
+          "who": "admin",
+          "organizationId": "Organization1",
+          "entityType": "Client",
+          "entityId": "Client1",
+          "entityVersion": "1.0",
+          "createdOn": 1445973207770,
+          "what": "Register"
+        }
+      ]
+    }
+  ]
+}
+----
+200
+Content-Type: text/plain;charset=utf-8
+
+INFO: ----------------------------
+INFO: Starting apiman data import: .*apiman_import_migrated.*.json
```

Comment:
I think there are some weird Windows line-endings in the original file that is causing the whole file to appear to be changed? If you can find a way of leaving that alone and only making your down change that would be great great -- but don't spend much time on it!